### ISSUE TITLE
Some details  in TTSR.py and SearchTransfer.py

### ISSUE THREAD

**Issue by lmybaobao** [researchmm/TTSR#21]
Thank you for sharing your great work!
I don't understand why  the lrsr, refsr and ref have to add 1 and then divide by 2 in TTSR.py? Is it a normalization?
`_, _, lrsr_lv3  = self.LTE((lrsr.detach() + 1.) / 2.)`
and why the T_lv3, T_lv2 and T_lv1 got by F.fold have to divide by (3*3) in SearchTransfer.py?
`T_lv3 = F.fold(T_lv3_unfold, output_size=lrsr_lv3.size()[-2:], kernel_size=(3,3), padding=1) / (3.*3.)`

**Comment by FuzhiYang** [comment]
1. For add 1 and then divide by 2, because lrsr is in [-1, 1] range, but the LTE's input are in [0, 1] range. So this step is to adapt range.
2. It is because x != fold(unfold(x)). fold(unfold(x)) will be (kernel size * kernel size) times of x

**Comment by lmybaobao** [comment]
>     1. For add 1 and then divide by 2, because lrsr is in [-1, 1] range, but the LTE's input are in [0, 1] range. So this step is to adapt range.
> 
>     2. It is because x != fold(unfold(x)). fold(unfold(x)) will be (kernel size * kernel size) times of x

Thanks for your reply!
